src/main/scala/com/gu/newsletterlistcleanse/NewsletterAWSCredentialProvider.scala

```diff
@@ -0,0 +1,8 @@
+package com.gu.newsletterlistcleanse
+
+import com.simba.athena.amazonaws.auth.{ AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain }
+import com.simba.athena.amazonaws.auth.profile.ProfileCredentialsProvider
+
+class NewsletterAWSCredentialProvider extends AWSCredentialsProviderChain(
+  new ProfileCredentialsProvider("ophan"),
```

Comment:
This is while we work locally, then we'll either have to credential providers, or one that is able to access both accounts.